### DISCUSSION TITLE
FreeBSD 13.1 does not comes up after resume with a message "Guest has not initialized the display (yet).

### DISCUSSION THREAD

**Discussion by samm-git** [utmapp/UTM#4565] [UTM Help (macOS)]
**Describe the issue**
FreeBSD 13.1 does not comes up after resume with a message "Guest has not initialized the display (yet). 

To reproduce: 

1. Install FreeBSD 13.1 
2. Run it and press "Pause"
3. Close the vm window
4. Open it again. It will show "Guest has not initialized the display (yet). At the same time OS is alive, e.g. i can type reboot command and OS would reboot at some time. It does not happens in text-only mode.

If step 3 is not done - resume works as intended. 

**Configuration**
* UTM Version: Version 3.2.4 (58)
* OS Version: 12.4 (21F79)
* Intel or Apple Silicon: Apple Silicon

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the top of the `QEMU` page is `Debug Log`. Turn it on and save the VM. After you experience the issue, open the VM settings again and select `Export Log...` and attach it here.

[debug.log](https://github.com/utmapp/UTM/files/9004846/debug.log)

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since Github has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

[config.plist.txt](https://github.com/utmapp/UTM/files/9004863/config.plist.txt)

**Comment by aesophor** [comment]
This happens to me with Arch Linux as well.

**Comment by osy** [comment]
On the latest UTM, add a serial port and you’ll be able to see more info

**Reply by inflation** [reply]
The serial terminal actually worked fine. Only the window failed to resume. I'm on Fedora without a GUI.

**Comment by samm-git** [comment]
Just verified - serial device on FreeBSD works fine after sleep, but display is not.
